build-docs.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * build-html.javascript
+ * This program will copy/convert files in /docs/ to /out/docs/
+ */
+/*jslint node*/
+"use strict";
+let directoryOutDocs;
+let fs;
+let head;
+let marked;
+let path;
+let tail;
+// chdir to /docs/
+process.chdir("docs");
+// require modules
+fs = require("fs");
```

Comment:
```suggestion
let fs = require("fs");
```
there's no reason to separate the declaration from the initialization (and doing so deviates from the overwhelming convention in the community). same throughout.

Reply:
ok, applied suggested styleguide in followon commit b7ae15d56